drbd/defaults.yaml

```diff
@@ -1,15 +1,13 @@
 drbd:
-  install_packages: true
-  with_ha: true
+  install_packages: false
+  with_ha: false
+  promotion: "drbd-node1"
+  res_template: "res_single_vol_v9.j2"
```

Comment:
The same with the template. I wouldn't put a default template name. 
I don't really know how drbd is used normally, but I would say that there is not a "default" way. It's perfect to provide the templates, I like that, but I would prefer that the user must to choose it explicitily.

Reply:
Agree! Thanks for the opinion.